docs-source/spring/content/platform/intellij-plugin/_index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Oracle Backend for Spring Boot and Microservices IntelliJ Plugin"
+description: "IntelliJ Plugin for Oracle Backend for Spring Boot and Microservices"
+keywords: "intellij plugin ide springboot spring development microservices development oracle backend"
+---
+
+## GA 1.3.0 - September, 2024
+
+Oracle Backend for Spring Boot and Microservices (OBaaS) is an IntelliJ plugin to browse, deploy, and modify workloads on the Oracle Backend for Spring Boot and Microservices (OBaaS) platform.This plug-in implements the functionalities available in the [OBaas CLI](../../development/cli), simplifying  access to Oracle Backend for Spring Boot and Microservices deployments from an IntelliJ IDE. 
+
+The plug-in allows access to the OBaaS platform services, including the Grafana, Spring, APISIX, Eureka and Jaeger web admin consoles. Additionally, you may create and bind workloads to your  Oracle Backend for Spring Boot and Microservices database deployment. Users may inspect OBaaS deployment namespaces, workloads, and related configuration.
+
+See the Oracle Free Use Terms and Conditions [License](https://oss.oracle.com/licenses/upl/)
+
+## Prerequisites
+
+* An operational OBaaS deployment, as configured through platform [setup](../../setup/).
```

Comment:
we should try to avoid using the abbreviated name (OBaaS) in the docs, it should always be the full name